src/lib/formats/tzx_cas.cpp

```diff
@@ -899,6 +1034,13 @@ static cassette_image::error cdt_cassette_load( cassette_image *cassette )
 	blocks = nullptr;
 	return err;
 }
+static cassette_image::error tsx_cassette_load( cassette_image *cassette )
```

Comment:
Please keep function spacing consistent.

Reply:
Disculpa, lo he revisado y parece al mismo nivel, indicame por favor donde lo he hecho mal

Reply:
Excuse me, I have reviewed it and it seems at the same level, please tell me where I have done wrong